server/aws-lsp-partiql/src/server/hover-info/hoverDictionary.ts

```diff
@@ -0,0 +1,96 @@
+export const hoverDictionary = {
+    keyword: {
+        as: [
+            '### AS',
+            'The AS keyword is optionally used when renaming a table or column. Aliases provide users with the ability to reduce the amount of code required for a query.',
+            '',
+            'Visit [PartiQL Documentation](https://partiql.org/dql/as.html)',
```

Comment:
PartiQL is used by a bunch of AWS consoles which all have their own documentation pages on query syntax, they might not be happy to have links to the PartiQL website inside their console pages (instead of to AWS docs). It might be better to either not have links, or add a parameter to the server creation to allow users to hide the link.

Reply:
Removed the `url`.